### DISCUSSION TITLE
Github Discussion

### DISCUSSION THREAD

**Discussion by Adexifeoluwa** [community/community#135284] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

What is github discussions all about??

**Comment by Hepheoluwah** [comment]
GitHub Discussions is a collaborative communication forum for the community around an open source or internal project. Community members can ask and answer questions, share updates, have open-ended conversations, and follow along on decisions affecting the community's way of working.

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/135285#discussioncomment-10291801) for reference to why this discussion was closed and locked. Thanks!
